rbac/management/migrations/0027_auto_20201103_1956.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 2.2.4 on 2020-11-03 19:56
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("management", "0026_permission_description")]
+
+    operations = [
+        migrations.AlterField(
+            model_name="permission", name="description", field=models.TextField(default="", null=True)
```

Comment:
This null=True is required, or the permission creation will cause db to complain description field could not be null even though we have a default value ""